src/Shared/Core/DurableTaskVersionUtil.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Reflection;
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Utility class for generating the user agent string for the Durable Task SDK.
+/// </summary>
+public static class DurableTaskUserAgentUtil
```

Comment:
Why is this public and why is it in the shared code? Since this is only used by the `AzureManaged` package, shouldn't it go into that project?

Reply:
I moved it to the shared/azuremanaged